rs/nns/sns-wasm/src/canister_api.rs

```diff
@@ -8,11 +8,12 @@ pub trait CanisterApi {
     fn local_canister_id(&self) -> CanisterId;
 
     /// Create a canister on a subnet with cycles assigned to a given controller.
-    async fn create_canister(
+    async fn create_sns_canister(
```

Comment:
Other nit: I'd keep this "create_canister" since there's nothing SNS canister specific

Reply:
Done!